scripts/test-with-kind.sh

```diff
@@ -71,14 +71,23 @@ function install_controller {
 
 }
 
-function run_integration_test {
-       local __type=$1
-       local __vpc_id=$( vpc_id )
-       echo -n "running integration test type $__type ... "
-       ginkgo -v -r test/integration/$__type -- --cluster-kubeconfig=${KUBECONFIG} \
-               --cluster-name=${CLUSTER_NAME} --aws-region=${AWS_REGION} \
-               --aws-vpc-id=$__vpc_id
-       echo "ok."
+function run_integration_tests {
+  local __vpc_id=$( vpc_id )
+
+  for __type in ${INT_TEST_DIR}/*
+  do
+    case `basename $__type` in
+      test_app) # /test_app contains test app images
+        continue
+        ;;
+    esac
```

Comment:
using a switch statement here so that when we have specific tests like the one included here https://github.com/aws/aws-app-mesh-controller-for-k8s/pull/598 we can upgrade the controller with the feature sets required, like such:


```
sidecar_test)
    helm upgrade -i appmesh-controller config/helm/appmesh-controller etc etc
    ;;
```